### PR TITLE
Shallow Rendering

### PR DESCRIPTION
**Pull request by tamarau**
Hi, 

My first attempt to translate shallow rending. 

I've got a couple of things I'm not too sure about so would like to ask for feedback here :)
- I called shallow rendering 'Flaches Rendern'; rendern because I don't think there is a good German equivalent we use? I've seen 'Darstellung' but it's not what I would use when talking to other devs about rendering.
- Not sure if I can leave 'unit tests' and 'high-level API' as they are? I marked them in italics for now until that is solved. :)
- General question: how does the side menu get updated to the new title of the page?

Thanks ☺️

### CODE REVIEW

File: content/docs/addons-shallow-renderer.md [hunk 1]
```diff
@@ -1,68 +1,67 @@
...
-> We also recommend checking out Enzyme's [Shallow Rendering API](https://airbnb.io/enzyme/docs/api/shallow.html). It provides a nicer higher-level API over the same functionality.
+> Wir empfehlen, sich die [Shallow Rendering API](https://airbnb.io/enzyme/docs/api/shallow.html) von Enzyme anzusehen. Sie ermöglicht eine bessere *high-level API* mit denselben Funktionalitäten.
 
-## Reference {#reference}
+## Referenz {#reference}
 
 ### `shallowRenderer.render()` {#shallowrendererrender}
 
-You can think of the shallowRenderer as a "place" to render the component you're testing, and from which you can extract the component's output.
+Man kann sich shallowRenderer als den "Ort" vorstellen, in der man die Komponente testet, und von wo aus der Output der Komponente entnommen wird.
```
Comment (ph1p) [inline content/docs/addons-shallow-renderer.md]:
```suggestion
Du kannst dir den shallowRenderer als den "Ort" vorstellen an dem die zu testende Komponente gerendert wird, und du daraus die Ausgabe der Komponente entnehmen kannst.
```

File: content/docs/addons-shallow-renderer.md [hunk 2]
```diff
@@ -1,68 +1,67 @@
...
 
-`shallowRenderer.render()` is similar to [`ReactDOM.render()`](/docs/react-dom.html#render) but it doesn't require DOM and only renders a single level deep. This means you can test components isolated from how their children are implemented.
+`shallowRenderer.render()` ist ähnlich wie [`ReactDOM.render()`](/docs/react-dom.html#render), nur dass es keinen DOM benötigt und nur einen Level tief rendert. Das bedeutet, dass man Komponenten isoliert von der Art und Weise wie deren Kinder-Komponenten implementiert werden, testen kann.
```
Comment (ph1p) [inline content/docs/addons-shallow-renderer.md]:
```suggestion
`shallowRenderer.render()` ist ähnlich wie [`ReactDOM.render()`](/docs/react-dom.html#render), nur dass es kein DOM benötigt und nur einen Level tief rendert. Das bedeutet, dass du Komponenten abgegrenzt testen kannst, unabhängig davon wie die Kind-Komponenten implementiert sind.
```

File: content/docs/addons-shallow-renderer.md [hunk 3]
```diff
@@ -1,68 +1,67 @@
...
 
 ### `shallowRenderer.getRenderOutput()` {#shallowrenderergetrenderoutput}
 
-After `shallowRenderer.render()` has been called, you can use `shallowRenderer.getRenderOutput()` to get the shallowly rendered output.
+Nachdem `shallowRenderer.render()` aufgerufen worden ist, bekommt man mit `shallowRenderer.getRenderOutput()` den flachen gerenderten Output.
```
Comment (ph1p) [inline content/docs/addons-shallow-renderer.md]:
```suggestion
Nachdem `shallowRenderer.render()` aufgerufen wurde, kannst du dir mit `shallowRenderer.getRenderOutput()` die flach gerenderte Ausgabe zurückgeben lassen.
```

**Comment by ph1p** [comment]
> Hi,
> 
> My first attempt to translate shallow rending.
> 
> I've got a couple of things I'm not too sure about so would like to ask for feedback here :)
> 
> * I called shallow rendering 'Flaches Rendern'; rendern because I don't think there is a good German equivalent we use? I've seen 'Darstellung' but it's not what I would use when talking to other devs about rendering.

Hi Tamara!

First. Thanks for your PR ;)

Right now I think "Flaches Rendern" is ok. You have also added the English words so that everyone can understand them. 

> * Not sure if I can leave 'unit tests' and 'high-level API' as they are? I marked them in italics for now until that is solved. :)

I changed "high level" to "übergeordnet". Might sound a bit better. Let me know what you think. "Unit Tests" is fine.

> * General question: how does the side menu get updated to the new title of the page?

Just update the `nav.yml`

File: content/docs/addons-shallow-renderer.md [hunk 4]
```diff
@@ -1,68 +1,67 @@
...
 expect(result.type).toBe('div');
 expect(result.props.children).toEqual([
-  <span className="heading">Title</span>,
+  <span className="heading">Titel</span>,
   <Subcomponent foo="bar" />
 ]);
 ```
 
-Shallow testing currently has some limitations, namely not supporting refs.
+Zurzeit hat flaches Rendering die Einschränkung, dass `refs` nicht unterstützt werden.
```
[the remark this answers is not in the document]
Comment (tamarau) [reply content/docs/addons-shallow-renderer.md]: Changed it a little bit in my commit :)

**Comment by tamarau** [comment]
Hi Phil, 

Thanks for your feedback! Really appreciated :) 

I've committed your changes and updated nav.yml. I also changed 'Output' in the last line to 'Ausgabe', so it matches the previous sentence.

File: content/docs/addons-shallow-renderer.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (ph1p) [reply content/docs/addons-shallow-renderer.md]: You translate `Shallow testing` into `flaches Rendering`. Why?

[the remark this answers is not in the document]
Comment (ph1p) [reply content/docs/addons-shallow-renderer.md]: I have read the docs again and it makes no difference here between testing and rendering. So your translation doesn't lose any sense, so I think it's good. ;)

**Pull request merged**